src/cluster.c

```diff
@@ -5731,13 +5734,15 @@ NULL
         } else {
             addReply(c,shared.ok);
         }
-    } else if (!strcasecmp(c->argv[1]->ptr,"nodes") && c->argc == 2) {
-        /* CLUSTER NODES */
+    } else if (!strcasecmp(c->argv[1]->ptr,"nodes") && (c->argc == 2 || c->argc == 3))
```

Comment:
```suggestion
    } else if (!strcasecmp(c->argv[1]->ptr,"nodes") && (c->argc == 2 || (c->argc == 3 && !strcasecmp(c->argv[2]->ptr, "verbose"))))
```

Would rather `CLUSTER NODES FOO` not be treated the same as `CLUSTER NODES`

Reply:
Good point. I have incorporated it but apologies that I had to force-push since I accidentally amend the last commit.